Sources/dummy-swiftc/main.swift

```diff
@@ -0,0 +1,27 @@
+// This program can be used as `swiftc` in order to influence `-version` output
+
+import Foundation
+
+import class TSCBasic.Process
+
+let info = ProcessInfo.processInfo
+let env = info.environment
+
+if info.arguments.last == "-version" {
+    if let customSwiftVersion = env["CUSTOM_SWIFT_VERSION"] {
+        print(customSwiftVersion)
+    } else {
+        print("999.0")
+    }
+} else {
+    let swiftPath: String
+    if let swiftOriginalPath = env["SWIFT_ORIGINAL_PATH"] {
+        swiftPath = swiftOriginalPath
+    } else {
+        swiftPath = "/usr/bin/swiftc"
```

Comment:
Can we use `swiftc` which is more likely to work across multiple platforms?

Reply:
Yah, I can change that in a follow-up. 

IIRC, TSC's APIs always expect an absolute path, so need to figure out how to obtain a reasonable one myself.

Reply:
Actually thinking about just removing this for now. The existing test should never reach the `else`.